packages/frontend/webapp/src/components/FormConnexion/FormConnexion.vue

```diff
@@ -62,10 +61,32 @@ import { useNavigationStore } from "@/stores/navigation.store.js";
 const userStore = useUserStore();
 const navigationStore = useNavigationStore();
 
+// reason: why the user has been redirected to the connexion page ?
```

Comment:
préférer le français, désormais, pour les explications en commentaire

Reply:
Je respectais ce qui semblait être la norme dans ce fichier, cf `// stores and api`, mais c'est corrigé et noté pour les prochaines fois.